docs/modules/getting_started/pages/first_steps.adoc

```diff
@@ -10,36 +10,23 @@ A Spark application is made of up three components:
 - Driver: the driver starts the designated number of executors and removes them when the job is completed.
 - Executor(s): responsible for executing the job itself
 
-Create a file named `pyspark-pi.yaml` with the following contents:
-
-[source,yaml]
-----
-include::example$code/pyspark-pi.yaml[]
-----
-
-And apply it:
+Create a `SparkApplication`:
 
+[source,bash]
 ----
 include::example$code/getting_started.sh[tag=install-sparkapp]
 ----
 
 Where:
 
 - `metadata.name` contains the name of the SparkApplication
-- `spec.version`: the current version is "1.0"
-- `spec.sparkImage`: the docker image that will be used by job, driver and executor pods. This can be provided by the user.
+- `spec.version`: SparkApplication version (1.0). This can be freely set by the users and is added by the operator as label to all workload resources created by the application.
+- `spec.sparkImage`: the image by the job, driver and executor pods. This can be a custom image built by the user or an official Stackable image. Available official images can be listed in the Stackable https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%spark-k8s%2Ftags[image registry].
```

Comment:
```suggestion
- `spec.sparkImage`: the image used by the job, driver and executor pods. This can be a custom image built by the user or an official Stackable image. Available official images are listed in the Stackable https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%spark-k8s%2Ftags[image registry].
```